src/main/java/com/monumental/models/api/CreateMonumentRequest.java

```diff
@@ -0,0 +1,170 @@
+package com.monumental.models.api;
```

Comment:
I had to use this class to avoid the JSON deserializer attempting to create Model classes such as Tags, References, etc without all of the values being present.